heron/lib/widgets/BookmarksPanel.js

```diff
@@ -557,11 +609,13 @@ Heron.widgets.BookmarksPanel = Ext.extend(Heron.widgets.HTMLPanel, {
 			name: this.scName,
 			desc: this.scDesc,
 			layers: this.scvisibleLayers,
+			layersopacity: this.scvisibleLayersOpacity,

```

Comment:
You are right. Adding Layer objects would be a better solution. Because of the limited time to implement the suppport of opacity in bookmarks, I have chosen for this simple solution. So this would be new issue when redesigning the bookmarks. 

The same applies to Editing Layers Styles.